data/json/overmap/overmap_special/specials.json

```
@@ -6461,5 +6461,249 @@
+      { "point": [ 1, 6, 1 ], "overmap": "mil_base_2g1_north" },
+      { "point": [ 2, 6, 1 ], "overmap": "mil_base_3g1_north" },
+      { "point": [ 3, 6, 1 ], "overmap": "mil_base_4g1_north" },
+      { "point": [ 4, 6, 1 ], "overmap": "mil_base_5g1_north" },
+      { "point": [ 5, 6, 1 ], "overmap": "mil_base_6g1_north" },
+      { "point": [ 6, 6, 1 ], "overmap": "mil_base_7g1_north" },
+      { "point": [ 7, 6, 1 ], "overmap": "mil_base_8g1_north" },
+      { "point": [ 0, 7, 1 ], "overmap": "mil_base_1h1_north" },
+      { "point": [ 1, 7, 1 ], "overmap": "mil_base_2h1_north" },
+      { "point": [ 2, 7, 1 ], "overmap": "mil_base_3h1_north" },
+      { "point": [ 3, 7, 1 ], "overmap": "mil_base_4h1_north" },
+      { "point": [ 4, 7, 1 ], "overmap": "mil_base_5h1_north" },
+      { "point": [ 5, 7, 1 ], "overmap": "mil_base_6h1_north" },
+      { "point": [ 6, 7, 1 ], "overmap": "mil_base_7h1_north" },
+      { "point": [ 7, 7, 1 ], "overmap": "mil_base_8h1_north" },
+      { "point": [ 0, 8, 1 ], "overmap": "mil_base_1i1_north" },
+      { "point": [ 1, 8, 1 ], "overmap": "mil_base_2i1_north" },
+      { "point": [ 2, 8, 1 ], "overmap": "mil_base_3i1_north" },
+      { "point": [ 3, 8, 1 ], "overmap": "mil_base_4i1_north" },
+      { "point": [ 4, 8, 1 ], "overmap": "mil_base_5i1_north" },
+      { "point": [ 5, 8, 1 ], "overmap": "mil_base_6i1_north" },
+      { "point": [ 6, 8, 1 ], "overmap": "mil_base_7i1_north" },
+      { "point": [ 7, 8, 1 ], "overmap": "mil_base_8i1_north" },
+      { "point": [ 0, 9, 1 ], "overmap": "mil_base_1j1_north" },
+      { "point": [ 1, 9, 1 ], "overmap": "mil_base_2j1_north" },
+      { "point": [ 2, 9, 1 ], "overmap": "mil_base_3j1_north" },
+      { "point": [ 3, 9, 1 ], "overmap": "mil_base_4j1_north" },
+      { "point": [ 4, 9, 1 ], "overmap": "mil_base_5j1_north" },
+      { "point": [ 5, 9, 1 ], "overmap": "mil_base_6j1_north" },
+      { "point": [ 6, 9, 1 ], "overmap": "mil_base_7j1_north" },
+      { "point": [ 7, 9, 1 ], "overmap": "mil_base_8j1_north" },
+      { "point": [ 0, 10, 1 ], "overmap": "mil_base_1k1_north" },
+      { "point": [ 1, 10, 1 ], "overmap": "mil_base_2k1_north" },
+      { "point": [ 2, 10, 1 ], "overmap": "mil_base_3k1_north" },
+      { "point": [ 3, 10, 1 ], "overmap": "mil_base_4k1_north" },
+      { "point": [ 4, 10, 1 ], "overmap": "mil_base_5k1_north" },
+      { "point": [ 5, 10, 1 ], "overmap": "mil_base_6k1_north" },
+      { "point": [ 6, 10, 1 ], "overmap": "mil_base_7k1_north" },
+      { "point": [ 7, 10, 1 ], "overmap": "mil_base_8k1_north" },
+      { "point": [ 2, 8, 2 ], "overmap": "mil_base_3i2_north" },
+      { "point": [ 2, 8, 3 ], "overmap": "mil_base_3i3_north" },
+      { "point": [ 2, 8, 4 ], "overmap": "mil_base_3i4_north" }
+    ],
+    "locations": [ "field" ],
+    "connections": [ { "point": [ 3, -2, 0 ] } ],
+    "city_distance": [ 4, -1 ],
+    "city_sizes": [ 4, 12 ],
+    "occurrences": [ 0, 1 ],
+    "flags": [ "CLASSIC" ]
```

```suggestion
    "flags": [ "UNIQUE", "MILITARY" ]
```
I'd suggest making this much rarer than every 1 in 2 overmaps. The unique flag lets you tweak it more precisely, I've written it here for one in ten overmaps.

---

make sure you don't set occurrences 1st value to 0 if you use the unique flag. Or you'll end up with 0% chance in x to spawn as night outlined.  Did that with the isherwood special and had to fix it.